src/client/Microsoft.Identity.Client.Brokers/RuntimeBroker.cs

```diff
@@ -153,6 +156,8 @@ internal class RuntimeBroker : IBroker
             using (var core = new NativeInterop.Core())
             using (var authParams = WamAdapters.GetCommonAuthParameters(authenticationRequestParameters, _wamOptions.MsaPassthrough))
             {
+                AddPopParams(authenticationRequestParameters, authParams);
```

Comment:
Why not add it to `WamAdapters.GetCommonAuthParameters` ?

Reply:
will be added in next commit, still refactoring